templates/server/index.html

```diff
@@ -1,460 +1,484 @@
       </div>
     </div>
-  </div>
-
-  <div class="p-strip--light is-deep is-bordered">
-    <div class="row">
-      <div class="col-12">
-        <h2>{{ releases.lts.short_version }} LTS 新增功能</h2>
-        <ul class="p-list is-split">
-          <li class="p-list__item is-ticked">由 Canonical 提供到 2023 年的长期支持</li>
-          <li class="p-list__item is-ticked">可在所有主流架构上运行 - x86、x86-64、ARM v7、ARM64、POWER8 和 IBM s390x (LinuxONE)</li>
-          <li class="p-list__item is-ticked">占用空间更小、启动速度更快的全新 ubuntu-minimal 映像</li>
-          <li class="p-list__item is-ticked">通过 chrony 实现快速准确的时间同步</li>
-          <li class="p-list__item is-ticked">新的默认服务器安装程序 ISO 带来全新界面和更快的安装速度</li>
-          <li class="p-list__item is-ticked">支持 ZFS（非常适合服务器和容器的新一代卷管理/文件系统）</li>
-          <li class="p-list__item is-ticked">LXD 3.0 - 提供包括集群、Qos 和资源控制（CPU、内存、块 I/O/ 图形处理器和存储配额）等在内的多种 Linux 容器</li>
-          <li class="p-list__item is-ticked">更新了 LXD (v3.0)、DPDK (v17.11.1)、Postgresql (v10.3)、Libvirt (v4.0)、NGINX (v1.13)、Qemu (v2.11.1)、Docker (v17.03)、Puppet (v4.10)、MySQL (v5.7)、PHP (v7.2) 等</li>
-          <li class="p-list__item is-ticked">安装 snap 来支持简单的应用程序安装和发布管理</li>
-          <li class="p-list__item is-ticked">Linux 4.15 内核</li>
-          <li class="p-list__item is-ticked">获得 AWS、Microsoft Azure、Joyent、IBM、Google Cloud Platform 和 Rackspace 客户机认证</li>
-          <li class="p-list__item is-ticked">通过 netplan.io 轻松配置网络连接</li>
-        </ul>
-        <p><a class="p-link--external" href="https://wiki.ubuntu.com/BionicBeaver/ReleaseNotes">查看发布说明了解更多详情</a></p>
-      </div>
+</section>
+
+<div class="p-strip is-bordered">
+    <div class="u-fixed-width">
+      <h2>20.04 LTS新功能</h2>
+  <ul class="p-list is-split">
+    <li class="p-list__item is-ticked">由Canonical提供5年的支持至2025年，ESM扩展支持至2030年。</li>
+    <li class="p-list__item is-ticked">可在主流平台上运行——x86-64，ARM v7，ARM64，POWER8，POWER9，IBM s390x（LinuxONE），且将引入对RISC-V的基础支持。</li>
+    <li class="p-list__item is-ticked">为<a href="/aws">AWS</a> and <a href="/azure">Azure</a>所定制的云镜像——Ubuntu Pro，包含了安全加固，认证，内核热补丁(Kernel livepatch)以及其他功能。</li>
+    <li class="p-list__item is-ticked">Ubuntu Server Live安装助手现在在联网的情况下自动更新到最新版本并获取最新安全漏洞补丁。提供自动安装的基础支持。</li>
+    <li class="p-list__item is-ticked">SSH支持2次身份认证（2FA）</li>
+    <li class="p-list__item is-ticked">WireGuard<sup>®</sup>——具有现代加密技术且简单易用的创新VPN技术</li>
+    <li class="p-list__item is-ticked">AppArmor3使系统更安全</li>
+    <li class="p-list__item is-ticked">更为灵魂的的bootloader，可容许故障磁盘启动。</li>
+    <li class="p-list__item is-ticked">在Microsoft Azure上对IPv6的支持更好</li>
+    <li class="p-list__item is-ticked">支持Amazon Web Services (AWS)最新的Instance Metadata Service（IMDSv2）</li>
+    <li class="p-list__item is-ticked">长期支持的Linux 5.4内核支持最新的硬件，且包含最新的安全更新。</li>
+    <li class="p-list__item is-ticked">软件版本更新：QEMU（v4.2）， libvirt（v6.0）， PHP（v7.4），Ruby（v2.7），GCC（V9.3），Python（v3.8），MySQL（v8.0），NGINX（v1.17）</li>
```

Comment:
This line is too long and doesn't wrap. Is there a whitespace-character that can break this list?